prboom2/src/SDL/i_system.c

```diff
@@ -363,7 +372,7 @@ char* I_FindFileInternal(const char* wfname, const char* ext, dboolean isStatic)
     {NULL}, // current working directory
     {NULL, NULL, "DOOMWADDIR"}, // run-time $DOOMWADDIR
     {DOOMWADDIR}, // build-time configured DOOMWADDIR
-    {DSDAPWADDIR}, // build-time configured location of dsda-doom.wad
+    {NULL, DSDAPWADDIR, NULL, GetExecutableDir}, // build-time configured location of dsda-doom.wad
```

Comment:
I wouldn't have anything against *adding* a relative data directory, but please do not *remove* the ability to add an absolute one. I need this for the Debian package.

Reply:
Could you elaborate on your needs for the Debian package please ?

Would having a separate absolute version of that path be a solution ? For example:

In CMake:
```cmake
set(DSDAPWADDIR_ABSOLUTE "${CMAKE_INSTALL_PREFIX}/${DSDAPWADDIR}")
```

In config.h.cin:
```c
#cmakedefine DSDAPWADDIR "@DSDAPWADDIR_ABSOLUTE@"
#cmakedefine BIN_TO_PWAD_PATH "@DSDA_BIN_TO_PWAD_PATH@"
```

In SDL/i_system.c:
```c
{DSDAPWADDIR}, // absolute path to dsda-doom.wad's directory
{NULL, BIN_TO_PWAD_PATH, NULL, GetExecutableDir}, // relative path from the executable to dsda-doom.wad's directory
```

One thing that cannot be changed is having the CMake `DSDAPWADDIR` be a path relative to the install prefix since that is necessary for CPack to work properly.

Reply:
> Would having a separate absolute version of that path be a solution ?

Yes, as it was before.